Pre-flight check fails when checking for group membership of the user

Tried `crc start` without the bundle downloaded yet and got a panic error:

```
jeffmaury@localhost crc-0.85.0-linux-amd64]$ ./crc start
crc - Local OpenShift 4.x cluster
INFO  Checking if Virtualization is enabled       
INFO  Checking if KVM is enabled                  
INFO  Checking if Libvirt is installed            
INFO  Checking if user is part of libvirt group   
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x884e68]

goroutine 1 [running]:
github.com/code-ready/crc/pkg/crc/preflight.preflightCheckSucceedsOrFails(0x9a6300, 0x9bb8f8, 0x9ab201, 0x29, 0x0)
	/home/prkumar/work/github/crc/pkg/crc/preflight/preflight_common.go:26 +0x188
github.com/code-ready/crc/pkg/crc/preflight.StartPreflightChecks()
	/home/prkumar/work/github/crc/pkg/crc/preflight/preflight_linux.go:25 +0x201
github.com/code-ready/crc/cmd/crc/cmd.runStart(0xddadb8, 0x0, 0x0)
	/home/prkumar/work/github/crc/cmd/crc/cmd/start.go:41 +0xce
github.com/code-ready/crc/cmd/crc/cmd.glob..func6(0xdb3b00, 0xddadb8, 0x0, 0x0)
	/home/prkumar/work/github/crc/cmd/crc/cmd/start.go:25 +0x3f
github.com/spf13/cobra.(*Command).execute(0xdb3b00, 0xddadb8, 0x0, 0x0, 0xdb3b00, 0xddadb8)
	/home/prkumar/work/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0xdb3640, 0xdb3fc0, 0xc00017df68, 0xc00017df88)
	/home/prkumar/work/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xdb3640, 0x1, 0xdb3fc0)
	/home/prkumar/work/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/code-ready/crc/cmd/crc/cmd.Execute()
	/home/prkumar/work/github/crc/cmd/crc/cmd/root.go:59 +0x2d
main.main()
	/home/prkumar/work/github/crc/cmd/crc/main.go:8 +0x20
[jeffmaury@localhost crc-0.85.0-linux-amd64]$ 

```

@gbraad I'm able to reproduce the issue with the user check with the latest crc (builded from master branch `ed1fd10`).

```
[fedora@tsedmik-cdk bin]$ ./crc start -b ~/Downloads/crc_libvirt_4.1.0-rc.5.tar.xz 
crc - Local OpenShift 4.x cluster
INFO  Checking if Virtualization is enabled       
INFO  Checking if KVM is enabled                  
INFO  Checking if Libvirt is installed            
INFO  Checking if user is part of libvirt group   
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x8c6a08]

goroutine 1 [running]:
github.com/code-ready/crc/pkg/crc/preflight.preflightCheckSucceedsOrFails(0x9fd100, 0xa14c40, 0xa026bc, 0x29, 0x0)
	/home/fedora/crc/pkg/crc/preflight/preflight_common.go:26 +0x188
github.com/code-ready/crc/pkg/crc/preflight.StartPreflightChecks()
	/home/fedora/crc/pkg/crc/preflight/preflight_linux.go:25 +0x201
github.com/code-ready/crc/cmd/crc/cmd.runStart(0xc00000dd80, 0x0, 0x2)
	/home/fedora/crc/cmd/crc/cmd/start.go:44 +0x4e
github.com/code-ready/crc/cmd/crc/cmd.glob..func6(0xe49e80, 0xc00000dd80, 0x0, 0x2)
	/home/fedora/crc/cmd/crc/cmd/start.go:30 +0x3f
github.com/spf13/cobra.(*Command).execute(0xe49e80, 0xc00000dd60, 0x2, 0x2, 0xe49e80, 0xc00000dd60)
	/home/fedora/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0xe499c0, 0x14, 0x0, 0x0)
	/home/fedora/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xe499c0, 0xc000197f68, 0x8c9a62)
	/home/fedora/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/code-ready/crc/cmd/crc/cmd.Execute()
	/home/fedora/crc/cmd/crc/cmd/root.go:57 +0x2d
main.main()
	/home/fedora/crc/cmd/crc/main.go:10 +0x25
```

I'm not a member of `libvirt` group:
```
[fedora@tsedmik-cdk bin]$ id fedora
uid=1000(fedora) gid=1000(fedora) groups=1000(fedora),4(adm),10(wheel),190(systemd-journal)
```

--- a/pkg/crc/machine/machine.go
+++ b/pkg/crc/machine/machine.go
@@ -103,15 +103,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		logging.InfoF(" Starting stopped VM ...")
 		host, err := libMachineAPIClient.Load(machineConfig.Name)
 		s, err := host.Driver.GetState()
 		if err != nil {
 			logging.ErrorF("Error getting the state for host: %v", err)
 			result.Error = err.Error()
 		}
+		if s == state.Running {
+			result.Status = s.String()
+			return *result, nil
+		}
 
 		if s != state.Running {
+			logging.InfoF(" Starting stopped VM ...")
 			if err := host.Driver.Start(); err != nil {
 				logging.ErrorF("Error starting stopped VM: %v", err)
 				result.Error = err.Error()

--- a/pkg/crc/preflight/preflight_checks_linux.go
+++ b/pkg/crc/preflight/preflight_checks_linux.go
@@ -2,7 +2,6 @@ package preflight
 
 import (
 	"bytes"
-	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
@@ -16,6 +15,7 @@ import (
 	"text/template"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine/libvirt"
 	"github.com/code-ready/crc/pkg/crc/oc"
@@ -172,10 +172,10 @@ func checkUserPartOfLibvirtGroup() (bool, error) {
 		return false, fmt.Errorf("%+v : %s", err, stdErr)
 	}
 	if strings.Contains(stdOut, "libvirt") {
-	        logging.Debug("Current user is already in the libvirt group")
+		logging.Debug("Current user is already in the libvirt group")
 		return true, nil
 	}
-	return false, err
+	return false, errors.NewF("%s not part of libvirtd group", currentUser.Username)
 }
 
 func fixUserPartOfLibvirtGroup() (bool, error) {

